classes/rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_reader :book, :person
```

Comment:
- [ ] Since `@book and @person` instance variables will be modified within the Rental class, it will be nice to also add setter methods for them. Something like so: 
``` 
 attr_accessor :date, :book, :person
```
Please kindly fix that.

Reply:
Hello @t-yanick. Do you mean to add these setters in the book and person files, right?

thanks